include/TrustWalletCore/TWPublicKey.h

```diff
@@ -38,6 +41,9 @@ TWData *_Nonnull TWPublicKeyData(struct TWPublicKey pk);
 TW_EXPORT_METHOD
 bool TWPublicKeyVerify(struct TWPublicKey pk, TWData *_Nonnull signature, TWData *_Nonnull message);
 
+TW_EXPORT_METHOD
+bool TWPublicKeyVerifyEd25519(struct TWPublicKey pk, TWData *_Nonnull signature, TWData *_Nonnull message);
```

Comment:
https://github.com/TrustWallet/wallet-core/pull/76
would expose a nicer C and C++ API

Reply:
Great, I'll update my PR.

Reply:
@ivan-trust Please don't, they were just reverted 😬 
@alejandro-isaza Is working on a different API, which will be released soon.